specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -1228,6 +1322,48 @@
                 }
             }
         },
+        "UpdateGatewaySettingsParameters": {
+            "properties": {
+                "restAuthCredential.isEnabled": {
+                    "type": "boolean",
+                    "default": true,

```

Comment:
Hi, we set `"default": true` here, but it seems that it does not take effect for C# generated code. Can someone give us suggestions on this issue?

Reply:
@shahabhijeet any suggestions here?

Reply:
Using `default` in swagger doesn't tell the client what to send by default, it tells the client that if they don't send the value, that it will assume the `default` value.

Your service has to assume `true` if the client doesn't send a value for that property.

Reply:
@fearthecowboy Thanks for your reply. Is it possible to specify default value? In this case, our service always needs `restAuthCredential.isEnabled` to be `true` now. But for compatibility reason we have to keep the property.